operator/config/resources/role_binding.yaml

```diff
@@ -8,5 +8,5 @@ roleRef:
   name: manager-role
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: aro-operator
   namespace: system
```

Comment:
is it intended namespace? (also in service account file)

Reply:
seems to be overlayed by the kustomize

Reply:
it gets set by kustomize (look in operator/config/default/kustomization.yaml)